ui/src/app/components/runtime-configuration-panel.tsx

```diff
@@ -392,73 +324,13 @@ const PanelMain = fp.flow(
         analysisConfig.diskConfig.detachable) ||
         (analysisConfig.computeType === ComputeType.Dataproc &&
           !analysisConfig.diskConfig.detachable));
-    // Casting to RuntimeStatus here because it can't easily be done at the destructuring level
```

Comment:
I guess we must have fixed this at some point, because it's already the right type before casting